guides/manual-installation.md

```diff
@@ -77,6 +79,8 @@ firewall-cmd --reload
 
 ### Install postgresql
 
+> Note: This guide was tested with the PostgreSQL server package version **_16-150500.10.3.2_** from the repository SLE-Module-Packagehub-Subpackages15-SP5-Updates. Using a different version of the postgresql-server package may require different steps or configurations, specially when changing the major number. Refer to the official [postgress documentation](https://www.postgresql.org/docs/) for further guidance
```

Comment:
We shouldn't promote a package version coming from package hub.
PostgreSQL is available in the base module `sle-module-basesystem`. We should check this.
And postresql-server is available in `sle-module-server-applications`